tests/h/views/admin_admins_test.py

```diff
@@ -3,23 +3,23 @@
 from __future__ import unicode_literals
 
 import mock
-from pyramid import httpexceptions
 import pytest
+from pyramid import httpexceptions
```

Comment:
Yes, although in practice my sorting algorithm is "whatever [isort](https://github.com/timothycrosley/isort) says".